tests/unit/data-table/DataTable.spec.js

```diff
@@ -1,5 +1,71 @@
+import { mount } from '@vue/test-utils'
 import DataTable from '@/components/data-table/DataTable.vue'
 
+import FfDataTableRow from '@/components/data-table/DataTableRow.vue'
```

Comment:
All these imports and the setting up of `global.components` is currently required as the dependencies are not explicitly set. All components are registered globally: https://github.com/flowforge/forge-ui-components/blob/5401ebad7b17be134f022cee02217f19baf85578/src/index.js#L24-L29